src/main/resources/templates/notifications.ftl

```diff
@@ -0,0 +1,57 @@
+[#import "macros.ftl" as macros]
+[@macros.renderHeader i18n.translate("section.notifications") /]
+
+[@macros.renderMenu i18n user /]
+<div class="container">
+    <h2>
+    ${i18n.translate("block.my-notifications.title")}
+        <div class="pull-right"></div>
+    </h2>
+[#if error?? && error?has_content]
+    <div class="alert alert-danger">
+    ${i18n.translate(error)}
+    </div>
+[/#if]
+    <table class="table table-bordered">
+        <tbody>
+[#if notifications??]
+    [#list notifications as notificationWithUser]
+        <tr>
+            [#if notificationWithUser.isRead(user)]
+            <td class="notification read">
+            [#else]
+            <td class="notification unread">
+            [/#if]
+
+            <a href="${notificationWithUser.URI}">
```

Comment:
I think we should alter this form so that it points to an endpoint that both marks the notification as read and redirects to the underlying URI. We can then remove the form below.